packages/github/src/probot.ts

```diff
@@ -201,10 +201,12 @@ export async function handleAction({
   context.log.info(`Completed`);
 
   async function loadConfig(): Promise<Config | undefined> {
-    const yamlConfig = await getGithubConfig(context, 'graphql-inspector.yml');
+    const identifier = 'graphql-inspector';
+    const yamlConfig = await getGithubConfig(context, identifier + '.yml');
```

Comment:
I think this should be `.yaml`

Reply:
Fixed in #216 and released already as `v1.18.0`

Reply:
@steebchen